### PR TITLE
Add initial database schema migrations

### LINKED ISSUE #64: Set up node database migrations

### PR DESCRIPTION
**Pull request by greenberga**
This contains a port of `scripts/create-database.js` to `node-pg-migrate`. I also added the start of a `docker-compose.yml` to simplify local testing.

***Before merging, let’s test this in staging!***

Closes #64

### CODE REVIEW

**Comment by greenberga** [comment]
If desired, I can add a commit that deletes `scripts/create-database.js` too 🙂

**Comment by alexbbt** [comment]
Deleting that file makes sense, but let's wait until everyone gets moved over to this new system

**Comment by alexbbt** [comment]
I made a new issue to handle deleting that file and connecting to the née Database #71

**Comment by alexbbt** [comment]
After further review, your PR seems to fully fulfill this, go ahead and delete that file and I'll close that issue

File: migrations/1608616875404_base.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+const notNull = true;
+
+exports.up = pgm => {
+    pgm.createTable('buildings', {
+	id: 'id',
```
Comment (alexbbt) [inline migrations/1608616875404_base.js:9]: The white space looks a little funny here

**Review by alexbbt — APPROVED** [review]
I would prefer if you updated the white space on the schema file to be normal indentation, otherwise it looks great!

**Comment by greenberga** [comment]
> I would prefer if you updated the white space on the schema file to be normal indentation, otherwise it looks great!

Going to follow up with a PR that formats all the JS files using Prettier, and adds it as a `devDependency` + a script in `package.json`.

**Review by alexbbt — APPROVED** [review]
Sounds good

File: package.json [hunk 2]
```diff
@@ -29,8 +29,20 @@
     "build": "netlify-lambda build src -c webpack.functions.js",
     "format": "prettier --write .",
     "precommit": "yarn format && yarn test",
-    "reset-db": "node scripts/create-database && node scripts/import-spreadsheet.js",
     "los": "node scripts/los.js",
+    "migrate": "node-pg-migrate --reject-unauthorized false",
```
Comment (olivernyc) [inline package.json:33]: We should eventually configure ssl properly! https://node-postgres.com/features/ssl

**Review by olivernyc — APPROVED** [review]

**Pull request merged**